### PR TITLE
Update scripts.markdown

### PR DESCRIPTION
**Pull request by JeeCee1**
Added note and link to template shorthand notation. Added note about quotation marks. This might be general yaml knowledge but due all the mixed examples I think it is good to state it here. Changed example to show use of "" and 2 conditions in a list at a single condition.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by JeeCee1 — COMMENTED** [review]
Added extra note about double and single quotation marks.

File: source/_docs/scripts.markdown [hunk 1]
```diff
@@ -657,7 +657,10 @@ automation:
 
 {% endraw %}
 
-`conditions` also accepts a [shorthand notation of a template condition][shorthand-template].
+`conditions` also accepts a [shorthand notation of a template condition][shorthand-template]. <br>
+It needs to be wrapped in quotation marks (```""```) unless using folding (```>```) where it must not use quotation marks. When wrapping in outside double quotation marks ensure any inner quotation marks are of the single (```'```) variant. <br>
```
Comment (frenck) [inline source/_docs/scripts.markdown:661]: - This is actually partially incorrect and not related to scripts at all, this is generic YAML stuff, which doesn't belong here.
- Do not use HTML in Markdown

File: source/_docs/scripts.markdown [hunk 2]
```diff
@@ -657,7 +657,10 @@ automation:
...
+Note: use ```conditions``` in contrast with ```condition``` when using [template shorthand notation](https://www.home-assistant.io/docs/scripts/conditions/#template-condition-shorthand-notation).
```
Comment (frenck) [inline source/_docs/scripts.markdown:662]: - This is incorrect and not full.
- Don't use bare links.

File: source/_docs/scripts.markdown [hunk 3]
```diff
@@ -657,7 +657,10 @@ automation:
 
 {% endraw %}
 
-`conditions` also accepts a [shorthand notation of a template condition][shorthand-template].
+`conditions` also accepts a [shorthand notation of a template condition][shorthand-template]. <br>
```
Comment (frenck) [inline source/_docs/scripts.markdown:660]: Avoid the use of HTML in Markdown.

File: source/_docs/scripts.markdown [hunk 4]
```diff
@@ -676,9 +679,9 @@ automation:
               - service: script.arrive_home
                 data:
                   ok: true
-          - conditions: >
-              {{ trigger.to_state.state == 'Home' and
-                 is_state('binary_sensor.all_clear', 'off') }}
+          - conditions: # defaults to 'and' logic
+              - "{{ trigger.to_state.state == 'Home'}}"
+              - "{{ is_state('binary_sensor.all_clear', 'off') }}"
```
Comment (frenck) [inline source/_docs/scripts.markdown:684]: I'm not sure why this is changed?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/scripts.markdown (hunk 1 above)
Comment (JeeCee1) [reply source/_docs/scripts.markdown:661]: Thanks for all the detailed respones!
 
We can argue that this is not a yaml tutorial site, but I spent quite some time figuring it all out. It does not come at a cost to include some info here. A generic page with all ins and outs of yaml (HA specific) with links from all doc pages would be very handy as well. Not all of us are yaml proficient.

The fact that, even after research, I seem to be incorrect (can you please state what is incorrect) shows that it needs explanation. Sometimes list, sometimes fold, sometimes single apostrophes, sometimes double apostrophes. Please look through the eyes of a new HA/yaml user.

File: source/_docs/scripts.markdown (hunk 2 above)
Comment (JeeCee1) [reply source/_docs/scripts.markdown:662]: - I'm just trying to update the docs with more info if I encounter struggles, to aid new users. I'm by no means proficient let alone an expert. What is incorrect?
- OK, new to me. I personally like it when it's clear from the link description but happy to change it.

File: source/_docs/scripts.markdown (hunk 3 above)
Comment (JeeCee1) [reply source/_docs/scripts.markdown:660]: Sorry, I read [here](https://www.markdownguide.org/basic-syntax/#line-break-best-practices) it's actually best practice for line breaks ?

File: source/_docs/scripts.markdown (hunk 1 above)
Comment (frenck) [reply source/_docs/scripts.markdown:661]: > We can argue that this is not a yaml tutorial site

We don't have to argue that. We are simply not.

File: source/_docs/scripts.markdown (hunk 3 above)
Comment (frenck) [reply source/_docs/scripts.markdown:660]: Not using them.

**Comment by frenck** [comment]
This PR is running stale; no new changes have been added after the code review a while back.  Therefore, going to close this PR for now. Feel free to re-open it when you are ready to work on it again.

../Frenck

**Pull request closed without merging**